include/variant_parser/variant_record.hpp

```diff
@@ -0,0 +1,193 @@
+#include <seqan3/io/stream/concept.hpp>
```

Comment:
What's the connection to the seqan3 VCF parser? Will you also be using it?

Reply:
Hi! So at the moment, seqan3 doesn't have a VCF parser implemented. So the one I'm implementing here will be used as a sort of framework for the future seqan3 implementation!

Reply:
I see, thanks for the explanation!